plugin/src/main/java/com/denizenscript/denizen/events/item/ItemRecipeFormedScriptEvent.java

```diff
@@ -56,29 +65,17 @@ public boolean matches(ScriptPath path) {
         return super.matches(path);
     }
 
-    @Override
-    public boolean applyDetermination(ScriptPath path, ObjectTag determinationObj) {
-        if (determinationObj.canBeType(ItemTag.class)) {
-            ItemTag result = determinationObj.asType(ItemTag.class, getTagContext(path));
-            event.getInventory().setResult(result.getItemStack());
-            return true;
-        }
-        else {
-            return super.applyDetermination(path, determinationObj);
-        }
-    }
-
     @Override
     public ScriptEntryData getScriptEntryData() {
         return new BukkitScriptEntryData(EntityTag.getPlayerFrom(event.getView().getPlayer()), null);
     }
 
     @Override
     public ObjectTag getContext(String name) {
-        switch (name) {
-            case "item": return result;
-            case "inventory": return InventoryTag.mirrorBukkitInventory(event.getInventory());
-            case "recipe": {
+        return switch (name) {
+            case "item" -> result;
+            case "inventory" -> InventoryTag.mirrorBukkitInventory(event.getInventory());
+            case "recipe" -> {
                 ListTag recipe = new ListTag();
                 for (ItemStack itemStack : event.getInventory().getMatrix()) {
                     if (itemStack != null && itemStack.getType() != Material.AIR) {

```

Comment:
Should be updated to the `ListTag` convertor constructor, something like `new ListTag(Arrays.asList(event.getInventory().getMatrix()), ItemTag::new)`